### ISSUE TITLE
overseer constantly restore deleted requests

### ISSUE THREAD

**Issue by zqbake** [sct/overseerr#3517]
### Description

Overseer continues to restore deleted request, resulting the movie deleted in both radarr and overseerr redownloaded, this happends endlessly and it's truely annoying.

### Version

1.33.0

### Steps to Reproduce

1. delete a request and movies in radarr.
2. wait
3. the request reshowed in the overseer request list and radarr collection. and also the downloading client.

### Platform

desktop

### Device

x86 docker container

### Operating System

archlinux

### Browser

chrome

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by danshilm** [comment]
Can you check for logs at the time of the movie being re-added and put them in a gist?
If it happened today, you can put all the logs for the day in a gist too, that's all good.

**Comment by Retalak** [comment]
I am willing to bet the requests that are automatically being re-added are Plex Watchlist requests. I had to disable automatic Watchlist requests for all of my users on 1.33 because of this issue.

**Comment by lmgpfr** [comment]
@danshilm Yeah i have this issue with plex watchlist, too. I manually verified, it was unmonitored in radarr some hours before:

2023-07-30T02:00:00.474Z [debug][Radarr Scan]: Title is unmonitored and has not been downloaded. Skipping item.{"title":"Nobody"}

2023-07-30T03:02:34.844Z [info][Watchlist Sync]: Creating media request from user's Plex Watchlist{"mediaTitle":"Nobody","userId":1}

2023-07-30T03:02:39.229Z [info][Radarr]: Found existing title in Radarr and set it to monitored.{"movieId":23,"movieTitle":"Nobody"}

Blame on Plex, not removing it from watchlist if watched/deleted, but anyway shouldnt readd here.

**Comment by lmgpfr** [comment]
Push

**Comment by burntcookie90** [comment]
`touch`


just because an issue has existed for awhile, doesnt mean it needs to be auto closed. this issue is still valid.

**Comment by dominikheiss** [comment]
Yeah, please find a solution for that...really annoys a lot of people. Especially with apps like "maintainarr" to keep your lib clean....
But if it just gets redownloaded, it plain sucks :/

**Comment by zhqu1148980644** [comment]
Push

**Comment by burntcookie90** [comment]
It's not stale, stale issues don't exist. Issues either exist or they're fixed.

**Comment by burntcookie90** [comment]
Not stale
